src/components/CheckboxGroup/CheckboxGroup.mdx

```diff
@@ -1,18 +1,22 @@
 import { ArgTypes, Canvas, Description, Meta, Source } from "@storybook/blocks";
-import { changelogData } from "./checkboxGroupChangelogData";
-import ComponentChangelogTable from "../../utils/ComponentChangelogTable";
 
-import * as CheckboxGroupStories from "./CheckboxGroup.stories";
+import ComponentChangelogTable from "../../utils/ComponentChangelogTable";
+import ComponentDocsHeader from "../../utils/components/ComponentDocsHeader";
 import Link from "../Link/Link";
+import * as CheckboxGroupStories from "./CheckboxGroup.stories";
+import { changelogData } from "./checkboxGroupChangelogData";
 
 <Meta of={CheckboxGroupStories} />
 
-# CheckboxGroup
+<ComponentDocsHeader
+  category="Form elements"
+  componentName="CheckboxGroup"
+  summary="Grouping of related options that allows a user to select one or more options"
```

Comment:
To be more specific and not use "options" twice, what about:

"Grouping of related checkboxes that allows a user to select one or more options"

or 

"Grouping of related inputs that allows a user to select one or more options"